src/resources/BreadcrumbResources.ts

```diff
@@ -1,34 +1,43 @@
-import { BreadcrumbIF } from '@/interfaces'
+import { BreadcrumbIF } from '@bcrs-shared-components/interfaces'
 import { Routes } from '@/enums'
+import { createPinia, setActivePinia } from 'pinia'
+import { useAuthenticationStore } from '@/stores'
+
+setActivePinia(createPinia())
+const authenticationStore = useAuthenticationStore()
 
 /** Returns URL param string with Account ID if present, else empty string. */
 function getParams (): string {
-  const accountId = JSON.parse(sessionStorage.getItem('CURRENT_ACCOUNT'))?.id
+  const accountId = authenticationStore.getCurrentAccount?.id || 0
   return accountId ? `?accountid=${accountId}` : ''
```

Comment:
Wonder if it would be a good idea to move these lines to the `getRegistryDashboardBreadcrumb` function and get rid of `getParams`.

Reply:
You're right,  `getParams()` is only used once here so normally I'd combine the code.

However, in Edit UI, `getParams()` is used twice, and in Create UI, it's used three times. For the sake of consistency, how about if we leave it like this for now?

Reply:
Deal, OK with me Sev. Resolving.